scripts/cover/coverGcov.sh

```diff
@@ -1,50 +1,46 @@
 #!/usr/bin/env bash
 set -e
-
 # set HOOT_HOME if not set
 if [ -z "$HOOT_HOME" ]; then
     HOOT_HOME=~/hoot
 fi
-
 # generate gcov files from gcda files
 cd $HOOT_HOME
-
+# create the gcov output folder
+if [ ! -d gcov ]; then
+    mkdir gcov
+else
+    rm -f gcov/*
+fi
 # hoot-core coverage
-mkdir gcovCore
-cd gcovCore
-gcov ../hoot-core/tmp/debug/*.gcda
+pushd hoot-core
+gcov -o $HOOT_HOME/hoot-core/tmp/debug/ $HOOT_HOME/hoot-core/tmp/debug/*.gcda
```

Comment:
Call `gcov` from the `$HOOT_HOME/hoot-core` directory to make it work correctly (without all of the `../` stuff).  All of the `.gcov` output files are in that folder.  They get updated with `grep` later and moved to the main output directory with everything else.